src/test/java/com/wikia/webdriver/common/templates/core/CoreTestTemplate.java

```diff
@@ -86,13 +86,23 @@ private void setPropertiesFromAnnotationsOnDeclaringClass(Class<?> declaringClas
 
   private void setPropertiesFromAnnotationsOnMethod(Method method) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "setPropertiesFromAnnotationsOnMethod" is 12 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)